Marlin/language_fr.h

```diff
@@ -199,14 +199,14 @@
   #define MSG_VB_JERK                       _UxGT("Vb jerk")
   #define MSG_VC_JERK                       _UxGT("Vc jerk")
 #else
-  #define MSG_VA_JERK                       _UxGT("Vx jerk")
-  #define MSG_VB_JERK                       _UxGT("Vy jerk")
-  #define MSG_VC_JERK                       _UxGT("Vz jerk")
+  #define MSG_VX_JERK                       _UxGT("Vx jerk")
+  #define MSG_VY_JERK                       _UxGT("Vy jerk")
+  #define MSG_VZ_JERK                       _UxGT("Vz jerk")
```

Comment:
it worked before , it's just confusion  with language , and if fr is dead ' en ' will work
If you look in ' en ' , it's my PR , 
And before this Pr ,  the 'en ' was used , and the ' fr ' is now the same as ' en '
No stress about it
These definition don't need to be in ' fr ' became same as ' en ' , but to be clean someone have added these
Thks

Reply:
Your starting-point is evidently out of date.

Reply:
`MSG_VX_JERK` is gone. Please remove.

If the French happens to match the English, the translations should still be included in the French language file so that we can use our script to see which strings are actually missing.

Reply:
Hey , i have difficulties to understand english , i know a little c++ , i don't understand github and i use only the NAVIGATOR to give my contribution
Don't ask me to make this ' fifth dimensions things ' , i don't know what are you talking about lollllllll
For these things ' The scripting of verification of the language unit ' ,  please make it by yourself or give me a video to understand what you say lolllllllllllllllllllllllll

I'm not a great engineer , but you can trust , all i make , I MAKE IT PERFECTLY
-Discover ' destination from ' bug inversion
-FWretract perfect behavior + slicer full compatible now ' long retract include '
-Fixes of pause system
-Extra fan speed

But all things that need high technicity , not for me .. sorry  ,  the script that check all language units , i don't know what is it 
Thanks